### ISSUE TITLE
Issue with "Compiling from Rust to WebAssembly": …

### ISSUE THREAD

**Issue by travisdoesmath** [mdn/content#505]
MDN URL: https://developer.mozilla.org/en-US/docs/WebAssembly/Rust_to_wasm

#### What information was incorrect, unhelpful, or incomplete?

when creating `package.json`, there is an instruction 

> Note that you need to fill in your own username, after the `@`, in the dependencies section.

but there is no `@` in the example `package.json`: 

```json
{
  "scripts": {
    "serve": "webpack-dev-server"
  },
  "dependencies": {
    "hello-wasm": "^0.1.0"
  },
  "devDependencies": {
    "webpack": "^4.25.1",
    "webpack-cli": "^3.1.2",
    "webpack-dev-server": "^3.1.10"
  }
}
```

Similarly, when instructed to create `index.js` later in the instructions, it says I need to fill in my npm username again, but it unclear where I need to fill in my username.

```js
const js = import("./node_modules/hello-wasm/hello_wasm.js");
js.then(js => {
  js.greet("WebAssembly");
});
```

> Note that you need to fill in your npm username again.

#### Specific section or headline?

[Using the package on the web](https://developer.mozilla.org/en-US/docs/WebAssembly/Rust_to_wasm#Using_the_package_on_the_web)

#### What did you expect to see?

Placeholder code

#### Did you test this? If so, how?

N/A

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/webassembly/rust_to_wasm`
* MDN URL: https://developer.mozilla.org/en-US/docs/WebAssembly/Rust_to_wasm
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/webassembly/rust_to_wasm/index.html
* Report started: 2020-12-20T21:01:06.302Z

</details>

**Comment by travisdoesmath** [comment]
Update: I was able to run the code in the tutorial without changing `package.json` or 'index.js` by slightly altering the following:

```
$ cd ../..
$ mkdir site
$ cd site
$ npm link hello-wasm
```
to

```
$ cd ..
$ mkdir site
$ cd site
$ npm link hello-wasm
```

It is unclear if `site` is meant to be included in the `hello-wasm` package. If so, I suggest changing the `cd ../..` line to `cd ..` and removing the instructions about adding usernames. Perhaps this is obvious to users with Node.js experience, but it is not to a relative novice like myself.

**Comment by MendyBerger** [comment]
1. _Note that you need to fill in your own username, after the @, in the dependencies section._ = **resolved in #2196**
2. _Note that you need to fill in your npm username again._ = **resolved in #4215**
3. `cd` to wrong directory = **resolved in #4215**


Think this issue is fully resolved now.

**Comment by MendyBerger** [comment]
1. _"Note that you need to fill in your own username, after the @, in the dependencies section."_ = **resolved in #2196**
2. _"Note that you need to fill in your npm username again."_ = **resolved in #4215**
3. `cd` to wrong directory = **resolved in #4215**
